src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/SqsEventListener.kt

```diff
@@ -22,7 +20,7 @@ class SqsEventListener(
     val log: Logger = LoggerFactory.getLogger(this::class.java)
   }
 
-  @JmsListener(destination = "\${sqs.queue.name}")
+  @JmsListener(destination = "whereabouts", containerFactory = "hmppsQueueContainerFactoryProxy")
```

Comment:
I don't _think_ we have existing integration tests which publishes a message to the LocalStack queue and validates it's processed, so will need to run validations in Dev/PreProd